src/sentry/features/temporary.py

```diff
@@ -76,6 +76,8 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:codecov-commit-sha-from-git-blame", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=False)
     # Enable continuous profiling
     manager.add("organizations:continuous-profiling", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
+# Enable continuous profiling
+    manager.add("organizations:continuous-profiling-allowed-ingest", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
```

Comment:
Can we name this `organizations:continuous-profiling-beta`? And leave `api_expose=True`? We'll need this flag for the frontend as well.